subsys/bluetooth/host/smp.c

```diff
@@ -1701,7 +1709,7 @@ static void smp_pairing_complete(struct bt_smp *smp, uint8_t status)
 
 	smp_reset(smp);
 
-	if (conn->sec_level != conn->required_sec_level) {
+	if (conn->state != BT_CONN_CONNECTED && conn->sec_level != conn->required_sec_level) {
```

Comment:
Shouldn't this be
```suggestion
	if (conn->state == BT_CONN_CONNECTED && conn->sec_level != conn->required_sec_level) {
```

Reply:
Absolutely, sorry. I copy pasted too quickly 😅